barretenberg/cpp/src/barretenberg/vm/tests/avm_execution.test.cpp

```diff
@@ -1319,6 +1319,69 @@ TEST_F(AvmExecutionTests, msmOpCode)
     validate_trace(std::move(trace), public_inputs, calldata, returndata);
 }
 
+// Positive test with pedersen commitment
+TEST_F(AvmExecutionTests, pedersenCommitmentOpcode)
+{
+    auto expected_result =
+        grumpkin::g1::affine_element(fr(uint256_t("054aa86a73cb8a34525e5bbed6e43ba1198e860f5f3950268f71df4591bde402")),
+                                     fr(uint256_t("209dcfbf2cfb57f9f6046f44d71ac6faf87254afc7407c04eb621a6287cac126")));
+    // grumpkin::g1::affine_eleelement;
+    // grumpkin::g1::affine_element b = grumpkin::g1::affine_element::one();
+
+    // Remmeber that grumpkin Fq == BN254 Fr => aka FF
+    grumpkin::g1::Fq scalar_a = grumpkin::g1::Fq::zero();
+    grumpkin::g1::Fq scalar_b = grumpkin::g1::Fq::one();
+    std::vector<FF> expected_output = { expected_result.x, expected_result.y, expected_result.is_point_at_infinity() };
+    // Send all the input as Fields and cast them to U8 later
+    std::vector<FF> calldata = { scalar_a, scalar_b };
+    std::string bytecode_hex = to_hex(OpCode::CALLDATACOPY) + // Calldatacopy...should fix the limit on calldatacopy
```

Comment:
What is this comment on a limit on calldatacopy?

Reply:
whoops a copy paste error